scripts/docker-testnet/variables.sh

```diff
@@ -114,32 +108,28 @@ export PORT_ORIGIN_VALIDATOR_REST="9500"
 
 # UI configuration profiles
 
-# Use tmux or not. If set to 1, only 2 terminal windows will be opened, and
-# tmux will be used to display the running executables using split windows.
-# Recommended. Tmux needs to be installed.
-export USETMUX=1
-
-# Log level for the logger in the Node.
-export LOGLEVEL="*:INFO"
-
-
-if [ "$TESTNETMODE" == "debug" ]; then
-  LOGLEVEL="*:DEBUG,api:INFO"
-fi
-
-if [ "$TESTNETMODE" == "trace" ]; then
-  LOGLEVEL="*:TRACE"
-fi
+## Use tmux or not. If set to 1, only 2 terminal windows will be opened, and
```

Comment:
commented code. Can use a local.sh file instead. We have exceptions for it in the .gitignore

Reply:
removed altogether. not needed in our context.